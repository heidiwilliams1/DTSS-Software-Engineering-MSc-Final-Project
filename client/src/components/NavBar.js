import React from "react";

class NavBar extends React.Component {
  render() {
    return (
      <div className="navBar">
        <a href=""> Pitch </a>
        <a href=""> Stats </a>
        <a href=""> Line-ups </a>
        <a href=""> Player Stats </a>
      </div>
    );
  }
}

export default NavBar;
