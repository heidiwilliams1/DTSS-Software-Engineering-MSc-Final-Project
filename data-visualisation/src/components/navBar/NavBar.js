import React from "react";

class NavBar extends React.Component {
  render() {
    return (
      <div className="navBar" data-testid="navbar">
        <a href=""> Pitch </a>
        <a href=""> Stats </a>
        <a href=""> Line-ups </a>
        <a href=""> Player Stats </a>
      </div>
    );
  }
}

export default NavBar;
