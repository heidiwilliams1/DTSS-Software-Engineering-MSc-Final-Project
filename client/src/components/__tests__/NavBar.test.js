
import {render} from '@testing-library/react';
import NavBar from "../NavBar";

describe("NavBar", () =>{
    it("renders correctly" , () => {
        const navbar = render(<NavBar />);
        expect(navbar).toMatchSnapshot();
    })
})