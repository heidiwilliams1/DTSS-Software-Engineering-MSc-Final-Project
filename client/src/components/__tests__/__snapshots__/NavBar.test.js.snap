// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="navBar"
      >
        <a
          href=""
        >
           Pitch 
        </a>
        <a
          href=""
        >
           Stats 
        </a>
        <a
          href=""
        >
           Line-ups 
        </a>
        <a
          href=""
        >
           Player Stats 
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="navBar"
    >
      <a
        href=""
      >
         Pitch 
      </a>
      <a
        href=""
      >
         Stats 
      </a>
      <a
        href=""
      >
         Line-ups 
      </a>
      <a
        href=""
      >
         Player Stats 
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
